lib/native/http.ts

```diff
@@ -28,6 +28,7 @@ export class HttpConnection extends NativeResourceMixin(BufferedEventEmitter) im
     protected constructor(native_handle: any) {
         super();
         this._super(native_handle);
+        this.on('error', () => { }); // suppress error throwing
```

Comment:
Node's typical [behavior](https://nodejs.org/api/events.html#events_error_events) is:
> If an EventEmitter does not have at least one listener registered for the 'error' event, and an 'error' event is emitted, the error is thrown, a stack trace is printed, and the Node.js process exits.

Users expect it to throw

Reply:
I've fixed this with new error handling, now that I understand the problem.